src/components/TransactionBlock/KlimaStakingTransactionBlock.tsx

```diff
@@ -298,10 +298,13 @@ const KlimaStakingTransactionBlock = ({
       })
 
       if (routeToKlima.items.length > 0) {
-        setSelectedRoute(mapRouteToOption(routeToKlima.items[0]));
-        setToolUsed(routeToUsdc.items[0].LiFiBridgeUsed ?? '');
+        // routeToKlima.items[0].estimate.fromAmount
```

Comment:
remove?

Reply:
Yeah replaced with the bestOfferIndex which by default returns 0 in case there's 1 offer/route.